package.json

```diff
@@ -21,10 +22,12 @@
     "query-string": "^4.3.1",
     "react": "16.0.0-alpha.6",
     "react-native": "0.44.2",
+    "react-native-actionsheet": "^2.2.0",
     "react-native-communications": "^2.2.1",
     "react-native-device-info": "^0.10.2",
     "react-native-elements": "^0.9.3",
     "react-native-htmlview": "^0.9.0",
+    "react-native-material-design-searchbar": "^1.1.4",
```

Comment:
I think I will try react-native-elements' search bar later. Another choice might be the react-native-search-bar.